src/Provider/3dTilesProvider.js

```diff
@@ -4,9 +4,35 @@ import PntsParser from '../Parser/PntsParser';
 import Fetcher from './Fetcher';
 import OBB from '../Renderer/ThreeExtended/OBB';
 import Extent from '../Core/Geographic/Extent';
-import { init3dTilesLayer } from '../Process/3dTilesProcessing';
+import { pre3dTilesUpdate, process3dTilesNode, init3dTilesLayer } from '../Process/3dTilesProcessing';
 import utf8Decoder from '../utils/Utf8Decoder';
 
+
+/**
+ * iTowns supports loading and displaying 3d-tiles tilesets.
+ *
+ *
+ * @module 3d-tiles
```

Comment:
* What is the structure of instanced objects for 3dTiles (THREE.js and itowns) and not the specification. 

for example in node tile :

```js
tileA  // (THREE.Mesh->itowns.tileMesh)
     {
          children: [tileB, tileC] // (Array) tile children
          parent: tile0  // (THREE.Mesh->itowns.tileMesh) tile parent 
          obb: // (itowns.OBB)  oriented bouding box
     }
```
in `3dTiles`, how are the objects linked: `content, parent, bounding Volume, region, transform ...`